src/components/Common/Module/FieldItem.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div>
```

Comment:
Any reason for not using flexbox and using floats instead ? If you resize the screen you'll see that it breaks (overlaps)

Reply:
It was a quicker solution for me. Changing float to flexbox now.